src/display/annotation_layer.js

```diff
@@ -2853,15 +2849,15 @@ class InkAnnotationElement extends AnnotationElement {
       polyline.setAttribute("stroke", "transparent");
       polyline.setAttribute("fill", "transparent");
 
-      // Create the popup ourselves so that we can bind it to the polyline
-      // instead of to the entire container (which is the default).
-      if (!popupRef && this.hasPopupData) {
-        this._createPopup();
-      }
-
       svg.append(polyline);
     }
 
+    // Create the popup ourselves so that we can bind it to the polyline
+    // instead of to the entire container (which is the default).
```

Comment:
I do believe that this comment is outdated now, and can simply be removed, since all annotations have a `getElementsToTriggerPopup`-method to find the relevant elements to bind the popup to.